### PR TITLE
Some recipe corrections

### PR DESCRIPTION
**Pull request by ocefpaf**
In this PR:

- added @pp-mo
- fixed the docs page URL
- moved from `ecmwf_grib` to `eccodes`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,30 +9,28 @@ source:
...
 requirements:
   build:
     - python
     - pip
-    - python-ecmwf_grib
   run:
     - python
-    - python-ecmwf_grib
+    - iris >=2
+    - python-eccodes
```
Comment (djkirkham) [inline recipe/meta.yaml:23]: Is there any way to specify "either package A or package B"? It will still work with python-ecmwf_grib.

**Review by djkirkham — APPROVED** [review]
Looks good to me.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:23]: I don't think conda can do that. We could remove both and let the user install them manually, not sure if that is a good idea though.

Comment (djkirkham) [reply recipe/meta.yaml:23]: No, let's use python-eccodes.

Comment (ocefpaf) [reply recipe/meta.yaml:23]: Another alternative would be to bump the build number and create another release with `python-ecmwf_grib` and let conda decided which one to pick based on what the users specifies.

Comment (djkirkham) [reply recipe/meta.yaml:23]: In that case it probably would be better to have the `python-eccodes` one having the higher build number. That way if the user doesn't specify either they will get eccodes, which is probably what we want.

**Pull request merged**